### PR TITLE
Server runtime

### PR DESCRIPTION
**Pull request by alexkuklin**
Related to #1052.

Defines runtime standalone server image with IDE part excluded.

#### Changelog

xsk-runtime images added in addition to xsk.

**New**

* xsk-runtime images added in addition to xsk.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -58,6 +58,13 @@ jobs:
           docker build --load -t ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk --build-arg JDK_TYPE=external-jdk --build-arg JDK_HOME=jdk .
           docker push ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk
           cd ../../
+      - name: Push XSK Runtime
+        run: |
+          cd releng/server-runtime
+          cp -r "${JAVA_HOME}" jdk
+          docker build --load -t ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk-runtime --build-arg JDK_TYPE=external-jdk --build-arg JDK_HOME=jdk .
+          docker push ${DOCKER_IMAGE_PREFIX:-dirigiblelabs/}xsk-runtime
```
Comment (krasimirdermendzhiev) [inline .github/workflows/build.yml:66]: For this step we will need new repository in dirigiblelabs  `dirigiblelabs/xsk-runtime`

**Review by krasimirdermendzhiev — APPROVED** [review]

**Review by krasimirdermendzhiev — APPROVED** [review]

File: releng/server-runtime/src/main/resources/dirigible.properties [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2017 SAP and others.
```
Comment (krasimirdermendzhiev) [inline releng/server-runtime/src/main/resources/dirigible.properties:2]: Need to change the license header

File: releng/server-runtime/src/main/webapp/403.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (krasimirdermendzhiev) [inline releng/server-runtime/src/main/webapp/403.html:1]: Need to change the license header

File: releng/server-runtime/src/main/resources/dirigible.properties (hunk 2 above)
Comment (alexkuklin) [reply releng/server-runtime/src/main/resources/dirigible.properties:2]: It is identical to `server` option.
I suppose it should be changed there as well then.

Comment (krasimirdermendzhiev) [reply releng/server-runtime/src/main/resources/dirigible.properties:2]: Yes! You can take from `sap-kyma-runtime`.

**Comment by dpanayotov** [comment]
moved to #1784

**Pull request closed without merging**